app/assets/javascripts/listings/components/listing/lottery-info-section.html.slim

```diff
@@ -25,9 +25,14 @@ section.content-wrap
         | {{ $ctrl.parent.listing.Lottery_Results_Date | date : 'longDate' }}
     p.c-steel.t-small translate="LOTTERY.COMPLETE_RESULTS_WILL_BE_POSTED"
 
+
   div ng-if="$ctrl.lotteryComplete()"
     .content-group.expand.text-center
       ng-include src="'listings/templates/listing/_lottery_results.html'"
+    .show-for-medium-up
```

Comment:
would be great to refactor the sidebar a bit, but am interested in working with our new designer to figure out how the pre-vs-post-lottery sidebar should really differ in content. For now I just added the waitlist info to the sidebar that we have post-lottery.